utils/service_discovery/sd_docker_backend.py

```diff
@@ -89,8 +89,11 @@ def __init__(self, agentConfig):
 
         self.docker_client = DockerUtil(config_store=self.config_store).client
         if Platform.is_k8s():
-            self.kubeutil = KubeUtil()
```

Comment:
Actually it could be an issue, we would have an error later accessing `self.kubeutil`